content/en/docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -346,6 +346,36 @@ service chaining and flexible load balancing.
 There are multiple, flexible ways to install JuniperContrail/TungstenFabric CNI.
 
 Kindly refer to this quickstart: [TungstenFabric](https://tungstenfabric.github.io/website/)
+{{% /tab %}}
+{{% tab name="Cilium" %}}
```

Comment:
It'd be good to list Cilium with other plugins alphabetically. (`JuniperContrail` wasn't, though)

Reply:
Ok, will move it.